docs/integrations/javascript/cloudflare.md

```diff
@@ -0,0 +1,77 @@
+---
+integration: logfire
+---
+
+# Cloudflare
+
+To instrument your Cloudflare Workers and send spans to Logfire, install the `@pydantic/logfire-cf-workers` and `@pydantic/logfire-api` NPM packages:
```

Comment:
do we need to specify which versions of Cloudflare Workers are supported?

Reply:
That's complicated to figure out, we're using a fork of the `otel-cf-workers` package, and I am not entirely sure if it has some minimum version requirements.